test/newwint.F90

```diff
@@ -0,0 +1,62 @@
+! This is a test for NCEPLIBS-bufr.
+!
+! This is a test for the newwin() subroutine.
+!
+! Ed Hartnett, J. Woollen, 3/15/2023
+program newwint
+  use moda_usrint
+  use moda_tables
+!  implicit none
+  integer idate, ins1, ins2, iret, iwin, jwin, lun, m, n
+  character*8 subset
+  integer msg_num, NUM_MSG
+  parameter(NUM_MSG = 12)
+  real(8) ncdt(5,50)
+  common /usrstr/ nnod,ncon,nods(20),nodc(10),ivls(10),kons(10)
+
+  print *,'Testing newwin()...'
+  
+  ! open a file 
+  open(unit = 20, file = 'testfiles/IN_2', form = 'UNFORMATTED')
```

Comment:
I added this fortran open call to the test. We need to open each file with Fortran before calling openbf(), right?

Also, how come this test, and all our other tests, doesn't call closbf() to close the file and release resources?

Reply:
When using openbf, then yes an open statement is needed prior to open the file and connect it to a logical unit number.  So thanks for catching that!

Many of our test codes do call closbf, including those which write/encode BUFR data.  But it's true that there are many which read/decode BUFR data (such as this) which don't, and that's also true of many application codes as well.  All of those are relying on the file being implicitly closed when the application itself terminates.